plugins/admin/pkg/command/autoscaling/update_test.go

```diff
@@ -36,22 +34,16 @@ func TestNewAsUpdateSetCommand(t *testing.T) {
 			},
 			Data: make(map[string]string),
 		}
-		client := k8sfake.NewSimpleClientset(cm)
-		p := pkg.AdminParams{
-			ClientSet: client,
-		}
-		cmd := NewAutoscalingUpdateCommand(&p)
+		p, _ := testutil.NewTestAdminParams(cm)
```

Comment:
@maximilien ```testutil.NewTestAdminParams``` is a helper function for writing UT codes, it will return a ```AdminParams``` object and ```kubenets client interface```, no **error** return for this function, in some UT functions, we won't use ```kubenets client interface```, that's why we ignore it. Here is the details:
```go
func NewTestAdminParams(objects ...runtime.Object) (*pkg.AdminParams, *k8sfake.Clientset) {
	client := k8sfake.NewSimpleClientset(objects...)
	return &pkg.AdminParams{
		NewKubeClient: func() (kubernetes.Interface, error) {
			return client, nil
		},
	}, client
}
```